Build warnings "extra ';'"

```
[ 28%] Building CXX object CMakeFiles/async_grpc.dir/async_grpc/server.cc.o
In file included from /home/chris/Development/async_grpc/async_grpc/rpc_handler.h:23:0,
                 from /home/chris/Development/async_grpc/async_grpc/server.h:30,
                 from /home/chris/Development/async_grpc/async_grpc/server.cc:17:
/home/chris/Development/async_grpc/async_grpc/rpc_service_method_traits.h:9:44: warning: extra ‘;’ [-Wpedantic]
                      const char* (*)(void));
                                            ^
/home/chris/Development/async_grpc/async_grpc/rpc_service_method_traits.h:11:56: warning: extra ‘;’ [-Wpedantic]
 DEFINE_HAS_MEMBER_TYPE(has_incoming_type, IncomingType);
                                                        ^
/home/chris/Development/async_grpc/async_grpc/rpc_service_method_traits.h:12:56: warning: extra ‘;’ [-Wpedantic]
 DEFINE_HAS_MEMBER_TYPE(has_outgoing_type, OutgoingType);
```

--- a/async_grpc/type_traits.h
+++ b/async_grpc/type_traits.h
@@ -42,7 +42,7 @@
                                                                                \
    public:                                                                     \
     static constexpr bool value = sizeof(check<U>(0)) == sizeof(std::uint8_t); \
-  };
+  }
 
 #define DEFINE_HAS_MEMBER_TYPE(traitsName, Type)           \
   template <class T>                                       \
@@ -61,7 +61,7 @@
    public:                                                 \
     static constexpr bool value =                          \
         sizeof(check<Derived>(0)) == sizeof(std::uint8_t); \
-  };
+  }
 
 #define DEFINE_HANDLER_SIGNATURE(traitsName, incomingType, outgoingType, \
                                  methodName)                             \

